stable/spinnaker/values.yaml

```diff
@@ -31,14 +31,14 @@ mail:
 
 # Images for each component
 images:
```

Comment:
I see, but it's really unrelated. Why not create a separate PR to make the version change explicit?